src/System.Linq.Expressions/src/System/Linq/Expressions/DefaultExpression.cs

```diff
@@ -21,9 +21,9 @@ internal DefaultExpression(Type type)
         }
 
         /// <summary>
-        /// Gets the static type of the expression that this <see cref="Expression" /> represents.
+        /// Gets the static type of the expression that this <see cref="Expression"/> represents.
         /// </summary>
-        /// <returns>The <see cref="Type"/> that represents the static type of the expression.</returns>
+        /// <returns>The <see cref="System.Type"/> that represents the static type of the expression.</returns>
```

Comment:
Good catch.  Did you find this, or is this something that was somehow caught by an analyzer?